proto/admin/v1/project.proto

```diff
@@ -44,4 +44,6 @@ message ProjectServiceRemoveProjectRequest {
 message ProjectServiceRemoveProjectResponse {
   // Success is a flag indicating if the member was removed successfully
   bool success = 1;
+  // Optional message to provide additional details
+  optional string message = 2;
```

Comment:
Isn't the error message descriptive enough ?

Reply:
I'm using this for success return in the api-server to show in the CLI, should I remove it?

```
message := fmt.Sprintf("Member %s successfully removed from project %s", req.MemberId, req.ProjectId)
	response := &adminv1.ProjectServiceRemoveProjectResponse{
		Success: true,
		Message: &message,
	}
```